### DISCUSSION TITLE
Getting error while starting first example

### DISCUSSION THREAD

**Discussion by Jcbtank** [OpenStaad/OpenStaadPython#2] [Q&A]
First of all thank you OpenstadAPI. I have started to learn python openstaad. below is the code and error i am getting while running the code.

from openstaad import Root
root =  Root()
file_name = root.GetSTAADFile()
print(file_name)

Error 

__new__
    p = type(_compointer_base)("POINTER(%s)" % new_cls.__name__,
             ^^^^^^^^^^^^^^^^
NameError: name '_compointer_base' is not defined. Did you mean: '_compointer_meta'?
PS C:\Users\jatin.tank>

**Comment by OpenStaad** [comment]
Hello I will take a look please give a more detailed information about the
issue.

Operating system
Staad license
Python version
If is posible a screen recording of the issue

Tk
